bson/_cbsonmodule.c

```diff
@@ -1790,7 +1790,7 @@ static PyObject* _cbson_dict_to_bson(PyObject* self, PyObject* args) {
     PyObject* result;
```

Comment:
We should probably null initialize every `PyObject*`

Reply:
Right, I left them alone since the initial goal was to silence Coverity and I'd be curious to see a Coverity run before setting more NULLs to make sure I didn't miss any. Also, would be nice to have some metric to indicate that setting the rest of them to NULL makes some difference somewhere … unless we already know this to be a positive change and don't care to measure.

Reply:
Fixed!